### ISSUE TITLE
One short note on source maps

### ISSUE THREAD

**Issue by dcodeIO** [SinaMFE/assemblyscript-typescript-loader#2]
Just so you are aware: Source map support currently requires that either the source map URL in the .wasm is absolute, or `compileStreaming` respectively `instantiateStreaming` are used. [[ref](https://github.com/WebAssembly/design/blob/d02d42fd40e3943bf0137927127ac8831952f15b/Web.md#linking-generated-code-to-source-maps)]

**Comment by happybang** [comment]
I can only  check the absolute URL for SourceMap is usefull , but the relative URL for sourceMap  is not usefull  in my firefox(v58.2) ，it's the same as  your [demo](https://github.com/AssemblyScript/assemblyscript/tree/master/examples/game-of-life)。it seems like unsupported by FF.

**Comment by dcodeIO** [comment]
Yes, relative source map URLs require the use of `compileStreaming` respectively `instantiateStreaming`.

**Comment by happybang** [comment]
i tried  `instantiateStreaming`  again like this:
```
 WebAssembly.instantiateStreaming(fetch(${publicPath}))
        .then(function(obj){
            return obj.instance.exports;
        });
```
but the FF still warn like this:

```
Sourcemap Error：TypeError: test.wasm.map is not a valid URL.
Source URL：wasm:http://192.168.1.106:3022/static/js/index.js:3317cec8b3f5c25c
SourcemapURL：test.wasm.map
```

is there same thing i ignore？
